Rendera/include/cube.hpp

```diff
@@ -41,7 +58,8 @@ class Cube
     vec3 m_position;
     vec3 m_a, m_b, m_c;
     Material m_material;
-    static const double m_epsilon = 1e-5;
+    vec3 int_pt;
+    const double m_epsilon = 1e-5;
```

Comment:
Why did you make this a non-`static` member? Now every `Cube` object will have its own copy of `m_epsilon`.